### ISSUE TITLE
Empty thumb tag in nfo causes Kodi poster search to skip local files

### ISSUE THREAD

**Issue by RobinQ124274** [Radarr/Radarr#2837]
I have been using Radarr for a week and I am totally loving it so far. However, I noticed Kodi was unable to find posters for my movies using the local files (`folder.jpg`, `poster.jpg` and `<movie>.jpg`) and instead displayed a random thumbnail of the movie. After inspection, I found out that there was an empty thumb tag (`<thumb />`) in `<movie>.nfo`. Upon removal of the empty tag, Kodi was again able to detect the local poster files.

Is the empty thumb tag intentional behavior? If so, is there a way to disable it within Radarr? I am currently using a script to correct the `<movie>.nfo` file during each import, but I suspect that the empty thumb tag shouldn't be there to begin with.

I am currently using version 0.2.0.1120 of Radarr on Linux and the following metadata settings:
- Emby: enabled (movie metadata)
- Kodi: enabled (movie metadata + movie images)
- Roksbox: disabled
- WDTV: enabled (movie metadata + movie images)

**Comment by RobinQ124274** [comment]
Upon further investigation, I noticed that the empty thumb tag was not the cause of the problem. Apparently the skin I am using in Kodi uses `<movie>.tbn` instead of the regular poster files to display the posters.

**Comment by mojo3141** [comment]
could we reopen?

I get same problem with standard skin - only non-standard setting I have is using mysql backend (advanced.xml).

fix should be simple - just need the nfo tags to point to the poster/fanart jpg that Radarr is generating.

(my workaround for now is just to disable nfo (meta) in Radarr, and let kodi handle things).

**Comment by RobinQ124274** [comment]
Sorry for the late reply, had a busy week ;). 

I did some digging and I found out that Radarr defaults to an empty `<thumb>` tag when no thumbnail files are found (as in frames from the video source). This can be found in the code, as displayed below:

[src/NzbDrone.Core/Extras/Metadata/Consumers/Xbmc/XbmcMetadata.cs](https://github.com/Radarr/Radarr/blob/develop/src/NzbDrone.Core/Extras/Metadata/Consumers/Xbmc/XbmcMetadata.cs)
```c#
var image = movie.Images.SingleOrDefault(i => i.CoverType == MediaCoverTypes.Screenshot);
...
if (image == null)
{
    details.Add(new XElement("thumb"));
}
else
{
    details.Add(new XElement("thumb", image.Url));
}
```

The Kodi documentation ([here](https://kodi.wiki/view/NFO_files/Movies)) mentions that both video thumbnails and posters can be declared by using the `<thumb>` tag, so it is indeed weird that Radarr only looks for thumbnail files and not the poster files.

Since this is a missing feature, I will reopen the issue. If I find some spare time in the weekend, I will try to load the project in Visual Studio, add the proper tags for posters and fanarts in the code and make a pull request. But I can't promise I will.
